Scanners/README.md

```diff
@@ -0,0 +1,27 @@
+# Scanners
+
+This category contains examples how to build your custom dependency scanner based on DBBs' `AbstractDependencyScanner` API.
+Samples are supposed to follow the same structure. The custom dependency scanner may either be a scanner implementation by itself or extend an existing scanner.
+
+## Building a scanner with IDZ
+
+* Clone the IBM/DBB repository into IDZ/Eclipse.
+* Import the scanner implementation as a JAVA project
+* Download the `<DBB-HOME>/lib/dbb.core_2.0.x.x.jar` package from your IBM DBB toolkit installation to your development environment 
```

Comment:
Maybe we should explicitly say "from your IBM DBB toolkit installation on z/OS Unix System Services to..."